Portal/Pages/Spaces/Details.cshtml

```diff
@@ -6,6 +6,21 @@
     ViewData["Title"] = $"Space {Model.SpacePageModel.Space.GetLastPathElementAsInt()}: {Model.SpacePageModel.Space.Name}";
 }
 
+@section header
+{
+    <link href="~/lib/dropzone-5.7.0/dist/min/dropzone.min.css" rel="stylesheet" />
+}

```

Comment:
The minified assets are available on CDN if you want to avoid source controlling the modules
https://www.jsdelivr.com/package/npm/dropzone?path=dist%2Fmin

Reply:
Will do